pkg/oci/load_balancer_util.go

```diff
@@ -505,3 +507,28 @@ func sortAndCombineActions(backendSetActions []Action, listenerActions []Action)
 	})
 	return actions
 }
+
+func getLoadBalancerSourceRanges(config *Config, service *api.Service) ([]string, error) {
+	var sourceRanges netsets.IPNet
+	var err error
+	if len(service.Spec.LoadBalancerSourceRanges) > 0 {
```

Comment:
Unsure whether this should have been completely removed from the GetLoadBalancerSourceRanges method in /kubernetes/pkg/api/v1/service/util.go